### PR TITLE
2 create navbar component

### LINKED ISSUE #2: Create Navbar Component
**Task title**
Create Navbar Component

**Task description**
Design, create, and implement a responsive Navbar component for the project, including navigation links and any necessary functionality.

**Subtasks**
- [x] Design the Navbar layout
- [x] Implement responsive design
- [x] Add navigation links

**Screenshots**

Image

**Link to the component on Figma**
https://www.figma.com/file/hhlhVG4NPMRH8VfwE17MkO/capstone-team-library?type=design&node-id=2321-4625&mode=design&t=X0wIFiJmCxZf78tw-4

### PR DESCRIPTION
**Pull request by DigitalDeveloperAM**
- Added the Navbar style and functionality for both language AR and EN.
- Added Logo according to figma design.
- Added Navigation Links to the Target pages.

- Added Test files according the project requirement and file Structure.
- All test running correctly.

# Related Issue

- Resolve #2

### CODE REVIEW

File: src/pages/index.jsx [hunk 1]
```diff
@@ -3,13 +3,16 @@ import { useTranslation } from "next-i18next";
 import { serverSideTranslations } from "next-i18next/serverSideTranslations";
 import * as React from "react";
 
+import Navbar from "@/components/Navbar";
```
Comment (Zino0031) [inline src/pages/index.jsx:6]: Can you remove this line

File: src/pages/index.jsx [hunk 2]
```diff
@@ -3,13 +3,16 @@ import { useTranslation } from "next-i18next";
...
+
```
Comment (Zino0031) [inline src/pages/index.jsx:7]: remove this line

File: src/pages/index.jsx [hunk 3]
```diff
@@ -3,13 +3,16 @@ import { useTranslation } from "next-i18next";
...
 import Layout from "@/layout/Layout";
 
 export default function HomePage() {
     const { t } = useTranslation("common");
 
     return (
         <Layout>
+            <Navbar />
```
Comment (Zino0031) [inline src/pages/index.jsx:15]: also remove this line

**Review by Zino0031 — CHANGES_REQUESTED** [review]
Hi Achref,

Great work on this pull request! However, I noticed that you've made some edits to index.js which we explicitly discussed not doing in order to avoid potential conflicts. Could you please remove the lines that you added to index.js and resubmit the changes?

Additionally, I'd like to remind you to request a review from us next time to notify us of your changes. This way, we can ensure that we're aware of and can provide feedback on the modifications made.

Thanks, and please let me know if you have any questions or need further clarification on the changes.

**Comment by melissasidisid** [comment]
great job my friend keep up the good working

**Review by melissasidisid — APPROVED** [review]
good work keep it up

**Review by HamidAhmidat — APPROVED** [review]
great job achraf, Im excited to see the navbar . well done.

**Pull request merged**